ml-agents/mlagents/trainers/trainer.py

```diff
@@ -268,3 +281,237 @@ def write_tensorboard_text(self, key, input_dict):
                 "Cannot write text summary for Tensorboard. Tensorflow version must be r1.2 or above."
             )
             pass
+
+
+class RLTrainer(Trainer):
```

Comment:
Does it make sense to make this its own file?

Reply:
Was debating this - I think it does, but I was also concerned with creating more files. Seems like it's long enough to break off, though.

Reply:
Done